doc/releases/release-notes-2.2.rst

```diff
@@ -429,7 +430,21 @@ Bluetooth
 
 * Host:
 
-  * <TBD>
+  * GAP: support for Extended Advertising
```

Comment:
We haven't added this. Is there a milestone somewhere that needs to be updated?

Reply:
Already removed, sorry